global-cli/index.js

```diff
@@ -167,6 +170,29 @@ function checkNodeVersion() {
   }
 }
 
+function checkAppName(appName) {
+  // TODO: there should be a single place that holds the dependencies
+  var dependencies = ['react', 'react-dom'];
+  var devDependencies = ['react-scripts'];
+  var allDependencies = dependencies.concat(devDependencies).sort();
+
+  if (allDependencies.indexOf(appName) >= 0) {
+    console.error(
+      chalk.red(
+        `Can't use "${appName}" as the app name because a dependency with the same name exists.\n\n` +
```

Comment:
This file can’t use template strings. It’s meant to be runnable in older nodes, at least until we warn about unsupported version.